.github/workflows/build.yml

```diff
@@ -34,9 +40,20 @@ defaults:
 env:
   BUILD_ID: ${{ inputs.buildId }}
   BASHBREW_ARCH: ${{ inputs.bashbrewArch }}
+
+  # the image we'll run to access the signing tool
+  # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
+  IMAGE_SIGNER: 'docker/image-signer-verifier:0.6.9@sha256:e38be2b9e6f010cf3432a2772b0e800feee572f7733c6df81e21293cb3e977e0'
+
+  # Docker Hub repository we'll push the (signed) attestation artifacts to
+  REFERRERS_REPO: oisupport/referrers
```

Comment:
There is only one of the iteration but it would be used more than once once we have the VSA in. It's not as obvious here but I wanted to do it like this to minimize the changrs